mod.go

```diff
@@ -71,7 +72,10 @@ func getVCSInfoForModule(m *listModule) (*moduleVCSInfo, error) {
 	if !ok {
 		return nil, errors.Newf("unknown VCS kind %q", root.VCS.Cmd)
 	}
-	dir := moduleDir(m.Path)
+	dir, _, err := moduleDir(m.Path)
```

Comment:
I think that ignoring the replDir return here is a mistake. It means that if we do:

   gohack get -vcs $module

when `$GOHACK` is relative, we'll have an absolute path in the replace directive instead of the relative path that we get (correctly) without the `-vcs` flag.

I'd suggest adding a `replDir` to the `moduleVCSInfo` type and using that to populate `modReplace.replDir` inside `updateVCSDir`.

Reply:
Yep again, this just makes this much less error prone. Thanks for the catch.